res/rules-engine/configuration.toml

```diff
@@ -74,6 +74,22 @@ PublishTopic="rules-events"
     Host = "localhost"
     Port = 6379
     Protocol = "redis"
+    [Trigger.EdgexMessageBus.Optional]
+    authmode = 'usernamepassword'  # required for redis messagebus (secure or insecure).
+    secretname = 'redisdb'
```

Comment:
same question about the case of key name

Reply:
Yes, the common code used expects them to be lower case. Same code is used for MQTT Export and MQTT trigger.

Reply:
got'cha, thx.